runtimes/alibi-explain/mlserver_alibi_explain/common.py

```diff
@@ -92,6 +91,10 @@ class Config:
 
     infer_uri: str
     explainer_type: str
+    explainer_batch: Optional[bool] = False
```

Comment:
Would it be worth opening an issue to document the Alibi runtime settings? We could use the same pattern we use for `Settings` and `ModelSettings` (i.e. leveraging some autodoc utils from Pydantic).

Reply:
Should we re-purpose https://github.com/SeldonIO/MLServer/issues/359 to add this kind of documentation?